stripe.go

```diff
@@ -20,10 +20,10 @@ const (
 )
 
 // apiversion is the currently supported API version
-const apiversion = "2014-12-17"
+const apiversion = "2015-01-11"
```

Comment:
Looks like these are the only changes made to the go bindings since the last release.